RELEASE_NOTES.md

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Need to add release version.

Reply:
Cool - will do

Reply:
BTW, as this is for PR purposes only, I will be collecting the data and then using it. I believe the release version goes into GitHub tool

Reply:
Yes. Agreed. The notes would be going in when we cut the release tag.